core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -149,6 +149,16 @@ public BigQuerySqlDialect(SqlDialect.Context context) {
   @Override public void unparseCall(final SqlWriter writer, final SqlCall call, final int leftPrec,
       final int rightPrec) {
     switch (call.getKind()) {
+    case CEIL_BIG_QUERY:
+      final SqlWriter.Frame ceilFrame = writer.startFunCall("CEIL");
+      call.operand(0).unparse(writer, leftPrec, rightPrec);
+      writer.endFunCall(ceilFrame);
+      break;
+    case FLOOR_BIG_QUERY:
```

Comment:
Can the unparse logic be pushed to the operator itself? Otherwise I think each dialect will have to override the `BIG_QUERY` style function calls (e.g. BQ incoming query targeting a Postgres backend).